lab1.js

```diff
@@ -74,6 +74,36 @@ var sentence2 = 'Come over here so you can scratch my belly.';
 
 //your code goes here
 
+//This functions takes a string parameter and changes all of the words to chirp
+var translateToMeerkat = function(sentence) {
+  //splits sentences
+  sentence = sentence.split(' ');
+
+  //changes each sentence array value to 'chirp'
+  for (var i = 0; i < sentence.length; i++) {
+  sentence[i] = 'chirp';
+  }
+
+  //strings array back to one string
+  var chirpString = '';
+  var chirpCount = 0;
+  while (chirpCount < sentence.length) {
```

Comment:
I like how you used both loops in the context of one solution (instead of two solutions, one loop for each solution).